### ISSUE TITLE
some unit tests are unhelpful

### ISSUE THREAD

**Issue by grahamgower** [popsim-consortium/stdpopsim#376]
I'm curious as to the value of certain tests found in the species test files. E.g. in `test_homo_sapiens.py`:
```
    def test_basic_attributes(self):
        self.assertEqual(self.species.population_size, 10**4)
        self.assertEqual(self.species.generation_time, 30)
```

And 
```
    def test_chromosome_lengths(self):
        genome = self.genome
        self.assertEqual(genome.get_chromosome("chr1").length, 249250621)
        self.assertEqual(genome.get_chromosome("chr2").length, 243199373)
        self.assertEqual(genome.get_chromosome("chr3").length, 198022430)
        ...
```

These exist for each of the species. As far as I can see, they only add maintenance burden, and are unlikely to catch any real bugs. I am in favour of removing these types of tests.

**Comment by grahamgower** [comment]
And to fan the smoke on this potential flame war, I'm going to leave this here:
https://rbcs-us.com/documents/Why-Most-Unit-Testing-is-Waste.pdf

**Comment by andrewkern** [comment]
I agree it's a bit of a maintenance burden, but a very small one and so I could take it or leave it. @jeromekelleher what's your take?

**Comment by jeromekelleher** [comment]
I see the point here @grahamgower, and I'm OK with removing if there's a consensus. Here's my take:

- I think ``test_basic_attributes(self):`` is useful because it gives us an insurance policy against people typoing these very important constants in the future. Suppose someone did a large refactor of the code, moving stuff all over the place and by accident added a zero the end of a ``population_size`` value. We might not spot such a bug for years. So, I don't see this as a maintenance burden, since we shouldn't be changing the basic API ever.

- The ``test_chromosome_lengths`` is less defensible. I added this because I didn't trust myself to copy the values from the NCBI website accurately (being a bit dyslexic with this sort of thing), and did it twice independently in different orders to make sure I did. If people are just copy-pasting the values, then I agree this is a bit pointless. We should probably just remove these tests (but have some strong guidance on how to import the necessary data).
